embulk-api/src/main/html/overview.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
why do we need this doc? How about having `package-info.java` instead?

Reply:
It's not very important, but just wanted to have some fundamental HTML elements from this in the top-level `index.html`. IIUC, `package-info.java` is for the package-level, not the top-level.